.changeset/ten-wolves-reflect.md

```diff
@@ -0,0 +1,13 @@
+---
+'@getodk/xforms-engine': minor
+---
+
+**BREAKING CHANGE**
+
+The main engine entrypoint (formerly `initializeForm`) has been split into:
+
+- `loadForm`, producing an intermediate result from which many instances of the same form can be created (with a `createInstance` method on that result)
+
+- `createInstance`, a convenience wrapper composing the result from `loadForm` and its `createInstance` method to create a single instance; this entrypoint effectively restores the behavior of `initializeForm`
+
+Some interfaces related to the former `initializeForm` have also been refined to reflect this change.
```

Comment:
The other changesets currently in the diff are part of #334 (and as I just mentioned there, I think the PR warrants a broader one as well). But I am pretty sure this changeset adequately covers the delta from that branch to this one!